addons/mail/static/tests/qunit_suite_tests/components/attachment_link_preview_tests.js

```diff
@@ -0,0 +1,119 @@
+/** @odoo-module **/
+
+import { insert, insertAndReplace } from '@mail/model/model_field_command';
+import {
+    start,
+    startServer,
+} from '@mail/../tests/helpers/test_utils';
+
+QUnit.module('mail', {}, function () {
+QUnit.module('components', {}, function () {
+QUnit.module('attachment_link_preview_tests.js');
+
+QUnit.test('auto layout with link preview', async function(assert) {
+    assert.expect(1);
+
+    const pyEnv = await startServer();
+    const irAttachmentId1 = pyEnv['ir.attachment'].create({
+        description: 'test description',
+        mimetype: 'application/o-linkpreview',
+        name: "https://tenor.com/view/gato-gif-18532922",
+        url: "https://tenor.com/view/gato-gif-18532922",
+    });
+    const mailChannelId1 = pyEnv['mail.channel'].create({});
+    pyEnv['mail.message'].create(
+        {
+            attachment_ids: [irAttachmentId1],
+            model: "mail.channel",
+            res_id: mailChannelId1,
+        }
+    );
```

Comment:
Format: `({` and `})` on same line. This saves an indent level, and also 2 LOCs.